backend/src/test/java/com/handong/rebon/review/domain/content/ReviewContentTest.java

```diff
@@ -0,0 +1,34 @@
+package com.handong.rebon.review.domain.content;
+
+import com.handong.rebon.exception.review.ReviewContentFormatException;
+import com.handong.rebon.exception.review.ReviewTitleFormatException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+
+class ReviewContentTest {
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @DisplayName("리뷰의 title은 null이거나 빈 문자열(공백만 있는 문자열 포함)이면 생성할 수 없다.")
+    public void reviewTitleIsNullOrEmptyException(String title) {
+        //given
+        //when, then
+
+        assertThatThrownBy(() ->new ReviewContent(title, "너무 좋아요"))
```

Comment:
```suggestion
        //given
        //when, then
        assertThatThrownBy(() -> new ReviewContent(title, "너무 좋아요"))
```
화살표와 new 사이 띄워주면 좋을 것 같네요